indico/web/client/js/utils/palette.js

```diff
@@ -36,7 +36,7 @@ export const SUIPalette = {
   purple: Translate.string('Purple'),
   pink: Translate.string('Pink'),
   brown: Translate.string('Brown'),
-  grey: Translate.string('Grey'),
+  gray: Translate.string('Gray'),
```

Comment:
```suggestion
  grey: Translate.string('Gray'),
```

The key can't be changed; it's one of the SUI colors.